### PR TITLE
{devel} [GCC-4.9.2] Clang 3.6 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
Not ready

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/d6c83d7be6ad9b47e609 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/1ba3156ebf5926b16d45 for a full test report.

**Comment by wpoely86** [comment]
@boegel @fgeorgatos ready!

File: easybuild/easyconfigs/c/Clang/Clang-3.6.0-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+sanity_check_paths = {
+    'files': ['bin/clang', 'bin/clang++', 'lib/libclang.so', 'lib/clang/%(version)s/include/stddef.h'],
+    'dirs': [],
+}
+
+assertions = True
+
+usepolly = True
+
+build_targets = ['X86']
```
Comment (boegel) [inline easybuild/easyconfigs/c/Clang/Clang-3.6.0-GCC-4.9.2.eb]: can you move these up, above `sanity_check_paths`?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a759d38f4f8659c0a721 for a full test report.

**Comment by wpoely86** [comment]
@boegel try again?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e14c2610824bb61a83fc for a full test report.

**Comment by boegel** [comment]
@wpoely86: several of the Clang tests are failing on my end...

```
  Expected Passes    : 15589
  Expected Failures  : 65
  Unsupported Tests  : 5036
  Unexpected Failures: 585
make[3]: *** [CMakeFiles/check-all] Error 1
```

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/8ca52ca073fcbb8327ea for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 5 out of 7 (1 easyconfigs in this PR)
Linux centos linux 7.0.1406, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/700a6a335b9a845cc83b for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
Linux centos linux 7.0.1406, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ebd0a4d70550dce43ac5 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
Linux centos linux 7.0.1406, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2209899d137079221e32 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2ba070170f2225378157 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.0.1406, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a28d97ac66d551c3b833 for a full test report.

**Comment by boegel** [comment]
@wpoely86: still fails for me, outside of a job (so with `ulimit -v unlimited`)

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux centos linux 7.0.1406, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1ce279445da430ee71c9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2b15415947424df263f7 for a full test report.

**Comment by wpoely86** [comment]
Success! :smile: 
So it works outside a PBS job and inside a PBS job only the sanitizer tests fail!

**Comment by boegel** [comment]
So, can we easily just disable those tests if we detect that we're in a PBS job? In the easyblock, that is?

**Comment by wpoely86** [comment]
hpcugent/easybuild-easyblocks#570 should do it!

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.0.1406, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/236befab98016fbd55cd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8dd25813a2c33e502fbe for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/21bca289393b258395dd for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/f21a3b48f805984e2f91 for a full test report.

**Comment by boegel** [comment]
Seems to be working now, on top of https://github.com/hpcugent/easybuild-easyblocks/issues/570, so going in.

Thanks @wpoely86!

**Pull request merged**